docs/common-patterns.rst

```diff
@@ -86,9 +86,7 @@ This is as opposed to the more intuitive sending pattern:
                 // Check if call succeeds to prevent an attacker
                 // from trapping the previous person's funds in
                 // this contract through a callstack attack
-                if (!richest.send(msg.value)) {
-                    throw;
-                }
+                require(richest.send(msg.value));
```

Comment:
I think this should be `assert()` here.

Perhaps we could also have a comment to say that the use of `richest.transfer(msg.value)` is preferred instead.